nautobot/ipam/models.py

```diff
@@ -898,6 +898,7 @@ def get_child_ips(self):
         Returns:
             IPAddress QuerySet
         """
+        # Question: This is a possible fix for this PR; should this be uncommented?
```

Comment:
?

Reply:
As discussed in standup, yeah, let's go with changing the behavior of `get_child_ips` since it's not used anywhere else that we're aware of.